mod-inn-reach/src/main/resources/volaris/features/inn-reach-location.feature

```diff
@@ -13,23 +15,125 @@ Feature: Inn reach location
+    * def notExistedLocationId = callonce uuid1
 
-  @Undefined
-  Scenario: Delete location
-    * print 'Delete location'
+  Scenario: Create inn reach location
+    * configure headers = headersUser
+    * print 'Create inn reach location 1'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "scdes",
+      "description": "Steelcase Design Library"
+     }
+    """
+    When method POST
+    Then status 201
+    * def locationId1 = $.id
+
+    * print 'Get central server by id 1'
+    Given path '/inn-reach/locations', locationId1
+    When method GET
+    Then status 200
+
+    * def locationResponseOne = $
+    And match locationResponseOne.code == "scdes"
+    And match locationResponseOne.description == "Steelcase Design Library"
 
-  @Undefined
-  Scenario: Get location by id
-    * print 'Get location by id'
+    * print 'Create inn reach location 2'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "plgen",
+      "description": "GVSU Pew Library General Collection"
+     }
+    """
+    When method POST
+    Then status 201
+    * def locationId2 = $.id
+
+    * print 'Get central server by id 2'
+    Given path '/inn-reach/locations', locationId2
+    When method GET
+    Then status 200
+
+    * def locationResponseTwo = $
+    And match locationResponseTwo.code == "plgen"
+    And match locationResponseTwo.description == "GVSU Pew Library General Collection"
 
-  @Undefined
   Scenario: Get locations
     * print 'Get locations'
+    * configure headers = headersUser
+    Given path '/inn-reach/locations'
+    When method GET
+    Then status 200
+    And match response.totalRecords == 2
 
-  @Undefined
-  Scenario: Create inn reach location
-    * print 'Create inn reach location'
+  Scenario: Check not existed location
+    * configure headers = headersUser
+    * print 'Check not existed location'
+    Given path '/inn-reach/locations', notExistedLocationId
+    When method GET
+    Then status 404
 
-  @Undefined
   Scenario: Update location
+    * configure headers = headersUser
     * print 'Update location'
+    Given path 'inn-reach/locations'
+    And request
+    """
+     {
+      "code": "ongen",
+      "description": "GVSU Steelcase Library General Collection"
+     }
+    """
+    When method POST
+    Then status 201
+    * def locationId = $.id
+
+    Given path '/inn-reach/locations', locationId
+    When method GET
+    Then status 200
+    * def locationResponseBefore = $
+    * set locationResponseBefore.code = "xxgen"
+    * set locationResponseBefore.description = "WV New Library General Collection"
+
+    Given path '/inn-reach/locations', locationId
+    And request locationResponseBefore
+    When method PUT
+    Then status 204
+
+    Given path '/inn-reach/locations', locationId
+    When method GET
+    Then status 200
+    * def locationResponseAfter = $
+    * set locationResponseAfter.code = "xxgen"
```

Comment:
not sure what you're testing here. shouldn't there be verification that updated values returned?

Reply:
Yes Here should be 
    And match locationResponseAfter.code == "plgen"
    And match locationResponseAfter.description == "GVSU Pew Library General Collection"